kickoff/jsonexportl10n.py

```diff
@@ -36,7 +36,7 @@ def l10nExport(releaseName):
 def _aggregateBetaLocales(releaseTable, releaseName):
     beta_prefix = releaseName.replace("beta", "b")
 
-    releases = releaseTable.query.filter(releaseTable.name.contains(beta_prefix))
+    releases = releaseTable.query.filter(releaseTable.name.contains(beta_prefix)).filter(releaseTable._shippedAt != None)
```

Comment:
I'm not sure if SqlAlchemy supports it, but would it be possible to use `is not None` instead?